lib/suspenders/generators/stylesheet_base_generator.rb

```diff
@@ -0,0 +1,23 @@
+require "rails/generators"
+
+module Suspenders
+  class StylesheetBaseGenerator < Rails::Generators::Base
+    source_root File.expand_path(
+      File.join("..", "..", "..", "templates"),
+      File.dirname(__FILE__))
+
+    def add_stylesheet_gems
+      gem "bourbon", "5.0.0.beta.5"
+      gem "neat", "~> 1.7.0"
+      gem "refills", group: [:development, :test]
+    end
+
+    def add_css_config
+      copy_file(
+        "application.scss",
+        "app/assets/stylesheets/application.scss",
+        force: true,
+      )
```

Comment:
Should we also do something if e.g. `app/assets/stylesheets/application.less` exists? And `app/assets/stylesheets/application.sass`? Why should we treat `.css` specially?

Thinking about this more, I'm tempted to issue no warning and not touch anything except `app/assets/stylesheets/application.scss`.

What do you think -- is that setting someone up for hours of annoying debugging?